interface-definitions/containers.xml.in

```diff
@@ -16,6 +16,16 @@
               <valueless/>
             </properties>
           </leafNode>
+          <leafNode name="bind-volume">
```

Comment:
Containers always talk about `volumes`, please rename this node.

Reply:
done